### PR TITLE
Updated the "Issue Tracker" URL in the index.html footer.

### PR DESCRIPTION
**Pull request by bseeger**
- Updated the "Issue Tracker" URL to point to Jira

Resovles: https://jira.duraspace.org/browse/FCREPO-1732

### CODE REVIEW

File: fcrepo-webapp/src/main/webapp/index.html [hunk 1]
```diff
@@ -37,7 +37,7 @@
 <nav class="navbar navbar-default navbar-fixed-bottom" role="navigation">
     <div class="container">
         <ul class="nav navbar-nav">
-            <li><a href="https://github.com/fcrepo4/fcrepo4/issues">Issue Tracker</a></li>
+            <li><a href="https://jira.duraspace.org/secure/IssueNavigator.jspa?reset=true&mode=hide&jqlQuery=project+%3D+FCREPO">Issue Tracker</a></li>
```
Comment (awoods) [inline fcrepo-webapp/src/main/webapp/index.html]: May I suggest this URL: https://jira.duraspace.org/browse/FCREPO

**Comment by bseeger** [comment]
What does the continuous integration test failure listed up above mean, specifically?  Did the test (or part of it) timeout?   Not sure what to do in response to this.  Thanks!

**Comment by awoods** [comment]
The travis-ci failure is probably an unrelated hiccup. In this case, no need to worry about it.

**Comment by awoods** [comment]
Resolved with: https://github.com/fcrepo4/fcrepo4/commit/98ec5e17554c663e93fcfedb18a44e196aa4b6f0

**Pull request closed without merging**